Formula/itstool.rb

```diff
@@ -19,14 +20,12 @@ class Itstool < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libxml2"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
 
   def install
-    xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    ENV.append_path "PYTHONPATH", "#{Formula["libxml2"].opt_lib}/python#{xy}/site-packages"
+    ENV.append_path "PYTHONPATH", Formula["libxml2"].opt_prefix/Language::Python.site_packages("python3")
 
-    system "./autogen.sh", "--prefix=#{libexec}",
-                           "PYTHON=#{Formula["python@3.9"].opt_bin}/python3"
+    system "./autogen.sh", "--prefix=#{libexec}", "PYTHON=#{which("python3")}"
```

Comment:
If `PYTHONPATH` isn't necessary we can probably install this straight into `prefix`. (Unless we're worried about conflicts, but I don't see why we would be.)